web_client/utils/analysis.jsx

```diff
@@ -141,14 +141,15 @@ export const runTask = (taskName, params, options = {}) => {
       .reduce(objectReduce, {})
   );
 
-  return (
-    rest({ path, type: 'POST', data })
-      .then(({
-        response: {
-          job: { _id: id }
-        }
-      }) => pollJob(id, title, taskName, pollInterval, 0, maxPolls))
-  );
+  return rest({ path, type: 'POST', data })
+    .then(({ response }) => response)
```

Comment:
What do you think about an access utility function ala Python's `operator.attrgetter` and `operator.itemgetter`?

```javascript
import pluck from './utils/...';

...

return rest({ path, type: 'POST', data })
    .then(pluck('response'))
```